scripts/configs/tr_args/default_tr_args.yaml

```diff
@@ -11,8 +11,8 @@ eval_strategy: "steps"
 # bf16: true
 save_steps: 500
 logging_steps: 10
-eval_steps: 50
-warmup_steps: 100
+eval_steps: 100
```

Comment:
Shouldn't this be overridden directly on ColQwen2's training config? That's not too important though, I don't mind!